### DISCUSSION TITLE
How can I test an agent without making a call with Twilio to avoid charges? I am encountering errors while running the quickstart client.

### DISCUSSION THREAD

**Discussion by orion-satyammishra** [voxos-ai/bolna#235] [Q&A]
Method to test an agent without incurring costs associated with making calls using Twilio. Additionally, it mentions facing errors while using the quickstart client.

**Comment by prateeksachan** [comment]
hey @orion-satyammishra Twilio offers trial accounts for testing purposes. 

What errors are you getting with the Twilio client?

**Reply by orion-satyammishra** [reply]
hi @prateeksachan 

Call is working fine with twilio, Is there any method already implement, through which  I can to have 1-1 audio conversation on my laptop, instead of doing having phone call.
